src/app/shell/item-comparators.ts

```diff
@@ -150,7 +150,21 @@ const ITEM_COMPARATORS: {
   // new -> old
   acquisitionRecency: acquisitionRecencyComparator,
   // None -> Kinetic -> Arc -> Thermal -> Void -> Raid -> Stasis
```

Comment:
I wonder if people expect stasis to come before kinetic. I really don't know.

Reply:
i don't feel like the dichotomy really has an order. stasis is also super weird and counts as elemental for some purposes, just as much as it doesn't. i feel ok about how this ends up sorted, and at least it's a no-op, compared to current.